tests/data/test_history.py

```diff
@@ -617,6 +629,16 @@ def test_datahandler_ohlcv_get_pairs(testdatadir):
     pairs = HDF5DataHandler.ohlcv_get_pairs(testdatadir, '5m')
     assert set(pairs) == {'UNITTEST/BTC'}
 
+    pairs = JsonDataHandler.ohlcv_get_pairs(testdatadir, '1h', 'mark')
+    assert set(pairs) == {'UNITTEST/USDT', 'XRP/USDT'}
+
+    # TODO-lev: The tests below
+    # pairs = JsonGzDataHandler.ohlcv_get_pairs(testdatadir, '8m')
+    # assert set(pairs) == {'UNITTEST/BTC'}
+
+    # pairs = HDF5DataHandler.ohlcv_get_pairs(testdatadir, '5m')
+    # assert set(pairs) == {'UNITTEST/BTC'}
+
```

Comment:
I don't know how to make a test HDF5 or Gz file for these tests

Reply:
i'm not sure we're at the test creation phase with this PR just yet ... as I'm not sure on usability (or how to make it usable) at the moment.

in theory, we can use something like `freqtrade download-data --trademode futures` (that parameter exists for list-pairs) ... but what should that download? 
only "mark" candles, or also futures candles?

my assumption is that futures candles might be slightly different to spot candles - so we'll really have 3 types of candles
* spot (for trading_mode=spot + margin)
* futures (trading_mode=futures)
* mark (trading_mode=futures)

as it won't make sense to have futures candles without mark candles (or viceversa) - I'd assume that it makes sense to download both at the same time (?)